src/extras/primitives/primitives.js

```diff
@@ -45,6 +45,7 @@ module.exports.registerPrimitive = function registerPrimitive (name, definition)
           var self = this;
           Object.keys(mappings).forEach(function resolveCollision (key) {
             var newAttribute;
+            if (key != key.toLowerCase()) console.warn('Mapping keys should be specified in lower case. This mapping may not be recognized');
```

Comment:
You can use `warn` https://github.com/morandd/aframe/blob/9e0f76f70927cbb9d1936f2dcd59f7d06654586d/src/extras/primitives/primitives.js#L9